File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Java/Exercises/21_compito_java_2018_04_06b/testaDiSerie.java
public class testaDiSerie extends Giocatore {
	private int ranking, eta;
	private boolean professionista;

	public testaDiSerie(int codiceGiocatore, String tipo, String nomeCognome, int ranking, int eta,
			boolean professionista) {
		super(codiceGiocatore, tipo, nomeCognome);
		this.ranking = ranking;
		this.eta = eta;
		this.professionista = professionista;
	}

	public String toString() {
		return nomeCognome + "\t" + codiceGiocatore + "\t" + tipo + "\t" + ranking + "\t" + eta + "\t" + professionista
				+ "\t-\t-\t-";
	}
}

File: II - Linguaggi di programmazione e laboratorio/Python/Exercises/21b_compito_java_20180406/TestaDiSerie.py
from Giocatore import Giocatore


class TestaDiSerie(Giocatore):
    def __init__(self, codice, tipo, nome_cognome, ranking, eta, professionista):
        super().__init__(codice, tipo, nome_cognome)
        self._ranking = ranking
        self._eta = eta
        self._professionista = professionista

    def __str__(self):
        return super().__str__() + "\t" + str(self._ranking) + "\t" + str(self._eta) + "\t" + str(self._professionista) + "\t-\t-\t-\t"
